game/scripts/npc/heroes/kunkka.txt

```diff
@@ -0,0 +1,13 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Kunkka
+  //=================================================================================================================
+  "npc_dota_hero_kunkka"
+  {
+    // Abilities
+    //-------------------------------------------------------------------------------------------------------------
+    "Ability10"     "special_bonus_attack_damage_90" //  replaces special_bonus_attack_damage_40
```

Comment:
Lmao no.

Reply:
This wont be as good as it looks.

Reply:
I suggest putting his torrent damage at level 10 making the number like 75 instead of 120 or whatever it is now, and then putting the 90 dmg at 15?

Reply:
I will do a talent shuffle on Kunkka.